docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,140 @@
+# Scalar DL Backup Creation and Restoration
+
+Scalar DL backup should be created after the Scalar DL server is paused otherwise there is a chance for inconsistency. 
+This guide shows you how to create and restore Scalar DL data backup without any data inconsistency.
+
+## Prerequisites
+
+* Read the Guide on How to [Back up and Restore Databases Integrated with Scalar DB](https://github.com/scalar-labs/scalardb/blob/master/docs/backup-restore.md)
+* Scalar DL must be deployed in the Kubernetes cluster
+* Cosmos DB account must be created with the backup policy `continuous` if you are using Cosmos DB.
+* You must synchronize time on the ledger and auditor servers if you are using the audit service.
+
+## Backup Creation
+
+This section shows how to create a transactionally-consistent backup for Scalar DL.
+
+### Requirements
+
+* You must wait at least 10 sec after pausing to create a backup.
```

Comment:
why 10 seconds?
It should be decided based on clock drift.

Reply:
Sorry for the inconvenience.
Requirements update like this
```
* You must wait at least 10 sec (based on clock drift) after pausing to create a backup.
    * You must be able to identify a common restore point between the pausing based on the clock drift of the Ledger and Auditor.
```